eng/generate-compiler-code.ps1

```diff
@@ -23,16 +23,17 @@ function Run-LanguageCore($language, $languageSuffix, $languageDir, $syntaxProje
   $errorFilePath = Join-Path $languageDir "Errors\$errorFileName"
   $errorGeneratedFilePath = Join-Path $generatedDir "ErrorFacts.Generated.$($languageSuffix)"
   $targetFramework = "net7.0"
+  $syntaxProjectTargetFramework = if ($language -eq "CSharp") { "net8.0" } else { $targetFramework }
```

Comment:
We can probably just update all the generators to be 8.0. Nothing keeping them on previous frameworks either.